tailwind-colors.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+import resolveConfig from "tailwindcss/resolveConfig";
+
+import tailwindConfig from "./tailwind.config";
+
+// $FlowIgnore
+const fullConfig = resolveConfig( tailwindConfig );
```

Comment:
What's the advantage of using `resolveConfig()` as opposed to importing from `tailwind.config.js` wherever we need it?

Reply:
When just importing tailwind.config.js seems like the colors from the config file could not be accessed in the code, at least dynamically. It would try to find the default colors from tailwind that didn't exist. 
This link to the docs explains a bit more: 
https://tailwindcss.com/docs/configuration#referencing-in-java-script

Reply:
Fair enough.